### PR TITLE
Made HW7, p.1-3.

### PR DESCRIPTION
**Pull request by anastasiya-harbatsevich**
p1. String Manipulator(remove characters from a given string)
p2. Email Validator
p3. Roman Numbers Calculator

### CODE REVIEW

File: src/main/java/org/example/homework7/EmailValidator.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class EmailValidator {
+    public static void main(String[] args) {
+        emailValidator();
+    }
+
+    public static void emailValidator() {
```
Comment (alexshamrai) [inline src/main/java/org/example/homework7/EmailValidator.java]: It is better to have method `isValidEmail(String email)` and call it from the main

File: src/main/java/org/example/homework7/RomanNumbersCalculator/RomanNumber.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package org.example.homework7.RomanNumbersCalculator;
+
+public enum RomanNumber {
+    I(1), IV(4), V(5), IX(9), X(10),
```
Comment (alexshamrai) [inline src/main/java/org/example/homework7/RomanNumbersCalculator/RomanNumber.java]: better to write everything in one line or every number in a new line
```suggestion
    I(1), 
    IV(4),
    V(5),
    IX(9),
    X(10),
```

**Review by alexshamrai — APPROVED** [review]

File: src/main/java/org/example/homework7/EmailValidator.java (hunk 1 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework7/EmailValidator.java]: Improved

File: src/main/java/org/example/homework7/RomanNumbersCalculator/RomanNumber.java (hunk 2 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework7/RomanNumbersCalculator/RomanNumber.java]: Improved

**Pull request merged**